Fix: Redirect on invalid instance

kaitmore opened the pull request:
Resolves #879

spmcbride1201 commented on src/components/Main/scenes/Fabric/FabricGrid.js:
```diff
@@ -41,18 +41,10 @@ class FabricGrid extends Component {
+    // State added by fabric router
+    // Display message if one is found on state
+    if (state && state.message) {
+      notification(state.message, {
```
Can you please refactor to use the reportError utility function?

spmcbride1201 commented on src/components/Main/scenes/Fabric/index.js:
```diff
@@ -53,15 +53,31 @@ function FabricRouter({ services }) {
-              : "";
-          // runtime informs the runtime-agnostic InstanceRouter which runtime router to render
-          // baseURL is prefixed to route paths and link to attributes when running with Fabric Server
+            services && serviceName && service ? service.runtime : "";
```
I'm thinking this could be a bit easier to understand if the variables are named to make it a bit more clear that they're booleans:
- userIsAuthorized
- serviceIsMetered
- instanceIsValid

spmcbride1201 commented:
Looking back at the issue, I think I mistyped. The behavior for this scenario should be to redirect to the service dashboard (showing the list of instances).

kaitmore commented:
@spmcbride1201 @karinakeen 

- [x] Added version # to error message
- [x] An invalid instance now redirects to the service dashboard instead of Fabric Grid
- [x] Refactored notifications to use reportError utility function
